python/utils/OpaqueArguments.h

```diff
@@ -212,10 +212,9 @@ inline void packArgs(OpaqueArguments &argData, py::args args) {
   }
 }
 
-/// @brief Return true if the given py::args represents a
-/// request for broadcasting sample or observe over all argument sets.
-/// Kernel arg types can be int, float, list, so
-/// we should check if args[i] is a list or ndarray.
+/// @brief Return true if the given `py::args` represents a request for
+/// broadcasting sample or observe over all argument sets. `args` types can be
+/// int, float, list, so  we should check if `args[i]` is a `list` or `ndarray`.
```

Comment:
nit: An argument could be made that int is not a word and is a code artifact. Same for float and list here.

Reply:
Updated in 9338e3fe5